App.js

```diff
@@ -44,26 +46,28 @@ module.exports = function App() {
         >
             <SafeAreaView style={{ flex: 1 }}>
                 <ThemeProvider theme={Theme}>
-                    <MiddleEnd.Provider middleEnd={middleEnd}>
-                        <ReactRedux.Provider store={middleEnd.store}>
-                            <NavigationContainer>
-                                <Stack.Navigator>
-                                    {Routes.map(({ path, component, options }) => {
+                    <ApplicationProvider {...Eva} theme={{ ...Eva.light, ...Theme }}>
```

Comment:
What's the relationship between the `ThemeProvider` and the `ApplicationProvider`? Does `ApplicationProvider` set the theme for child components whereas `ThemeProvider` allows us to access the theme within styled components e.g. stuff like `${({ theme }) => theme.spacing(2)}px;`?

Reply:
I saw Eva ships a [Material theme](https://www.npmjs.com/package/@eva-design/material). Is that not quite what we're going for style-wise? I'm still trying to sort out how all Eva's theming works and such, so my bad if this is a total whiff 😂

Reply:
Oh yea RE: your second question: we went with UI Kitten specifically because it was a little more iOS/Android agnostic, so I don't think we want to mess around with that Material theme.

Reply:
Ok so yea you're basically right on in your first question—`ApplicationProvider` provides the custom theme to UI Kitten's components, and `ThemeProvider` gives you `{ theme }` in styled components :+1:

Reply:
Ahh, ok, I think I got my brain scrambled and equated us wanting to make our styling API look like Material-UI's with using a Material Design theme :its-free: 

But cool, thanks for clarifying, all this is starting to click 🙏